Smaller updates to the NC docs

psatyal opened the pull request:
**Changelog:**

User guide: Added missing documentation on how to revoke OAuth access token provided by Nextcloud, via OpenProject.
System admin guide: Updated title to "Nextcloud integration setup"

psatyal committed:
Added details on revoking OAuth token

Added missing documentation on how to revoke OAuth access token provided by Nextcloud, via OpenProject.

psatyal commented:
Requesting review from @wielinde and @apfohl to confirm that technically, the text makes sense. I've tried to strike a balance between "technically correct" and relevant/understandable to the average user.

We will update the text on Andreas' PR targeting dev (https://github.com/opf/openproject/pull/11200) based on this text, so let's finalise this first.

Cheers
Parimal

psatyal commented:
https://community.openproject.org/work_packages/43644/activity

apfohl commented on docs/user-guide/nextcloud-integration/README.md:
```diff
@@ -120,12 +119,36 @@ When a Nextcloud file or folder is linked to a work package, an OpenProject user
+
+In certain cases, you may wish to disconnect your Nextcloud and OpenProject accounts. You might want to do this, for example, if you have multiple accounts and realise that you connected the wrong ones. 
+
+Since OAuth credentials were generated by both Nextcloud and OpenProject when you connected your accounts, we recommend disconnecting on both ends.
```
I don't think both steps are always necessary. But I see that it might be hard to find out which account is wrongly connected.

psatyal replied:
We discussed this with @wielinde yesterday, but perhaps there are easier ways to communicate this. The thing is, if you only disconnect on one end, we'll have to explain to the user what happens to the end where they didn't disconnect. (Or indeed, that information can still flow from oen side).

We can discuss this this afternoon during our call.

psatyal committed:
Typos

apfohl commented on docs/user-guide/nextcloud-integration/README.md:
```diff
@@ -120,12 +119,36 @@ When a Nextcloud file or folder is linked to a work package, an OpenProject user
+
+#### In OpenProject
+
+In OpenProject, click on your avatar on the top right corner and navigate to ***My account** → **Access tokens***. 
+
+![OpenProject account settings page](my_account_access_tokens.png)
+
+You will see a list of all access tokens associated with your account, including for a Nextcloud OAuth application. Click on **Revoke** and, in the confirmation dialog that appears, click **OK**.
```
So, this section will then change in my next PR I guess?

psatyal replied:
Exactly.

apfohl reviewed (approved):
Nothing major just comments that might need some thinking but nothing that needs changing immediately.

psatyal committed:
Updates

Updates:

- Nextcloud app is called "OpenProject Integration" (capital I)
- Updated title to include the word "setup"
- Updating the text of link to the user guide

wielinde commented:
@psatyal I am afraid we need to pause the work on the documentation for a moment. If I understand right, there is no way to revoke a token that was provided from Nextcloud to OpenProject. So, the information presented here in the docs are simply not correct. We need to fix that first and then come back to the text here. The work here is not lost.

psatyal commented:
Hi @wielinde. We want the docs to be reliable so yes, on hold for now. But if revoking the Nextcloud token is not yet possible, what happens when I go to _My account → Access tokens_ and click on on "**Revoke**" on the Nextcloud token?

PS: There's a part of the PR that I wanted to push immediately (update to title of the admin docs somehow didn't get pushed in my last PR), but I can submit another PR for that.

psatyal committed:
Remove disconnection instructions

Removing the added bit about removing NC access tokens in OpenProject until we have more details on it

psatyal commented:
Removing the bit about removing NC access tokens in OpenProject until we have more details on it. This PR now simply introduces some fixes.
